Makefile

```diff
@@ -56,6 +56,28 @@ ifneq ($(TAG),latest)
 BUNDLE_VERSION ?= --version=$(shell sed s/^v// <<< $(TAG))
 endif
 
+# To get the GIT commit id
+ifndef GIT_COMMIT
+GIT_COMMIT=$(shell git rev-list -1 HEAD)
+endif
+
+# Fetch the git tag
+ifndef GIT_TAG
+GIT_TAG=$(shell git describe --tags --exact-match --abbrev=0 2>/dev/null)
```

Comment:
Makefile syntax makes it possible to drop the `ifndef`/`endif` by using the `?=` operator:

```make
GIT_TAG ?= $(shell git describe --tags --exact-match --abbrev=0 2>/dev/null)
```

Reply:
replaced ifndef block with ?=